flake8_gl_codeclimate/__init__.py

```diff
@@ -115,7 +115,9 @@ def _violation_to_codeclimate_issue(cls, v):
             # issue, including deeper explanations and links to other resources.
             "categories": cls._guess_categories(v),
             "location": {
-                "path": v.filename,
+                # remove "./" at the beginning of the filename
+                # see: https://docs.gitlab.com/ee/ci/testing/code_quality.html#implementing-a-custom-tool
+                "path": v.filename[2:],
```

Comment:
Just to be on the safe side:
```suggestion
                "path": v.filename[2:] if v.filename.startswith("./") else v.filename,
```